apps/prairielearn/src/ee/pages/instructorAiGenerateDraftEditor/instructorAiGenerateDraftEditor.html.ts

```diff
@@ -73,13 +83,16 @@ export function InstructorAiGenerateDraftEditor({
                   >
                     <input type="hidden" name="__csrf_token" value="${resLocals.__csrf_token}" />
                     <input type="hidden" name="__action" value="regenerate_question" />
-                    <div class="form-group">
-                      <label for="user-prompt-llm">What needs to be changed?</label>
-                      <textarea name="prompt" id="user-prompt-llm" class="form-control"></textarea>
-                    </div>
-                    <button class="btn btn-primary">
+                    <textarea
+                      name="prompt"
+                      id="user-prompt-llm"
+                      class="form-control mb-2"
+                      placeholder="What would you like to change?"
+                      aria-label="Modification instructions"
+                    ></textarea>
+                    <button type="submit" class="btn btn-dark w-100">
```

Comment:
We use the word "change" in the AI response text and "what would you like to change?" but then we use "Adjust" for the button. Should we use the same term in both places? Maybe "Change question" would be simpler language for the button in any case?

Reply:
Reasonable point about consistency. I don't actually love the verb "change" here, I think I'd rather use something like "adjust" or "revise" (with "adjustments" or "revisions" to describe what happened after the fact). How does this sound?

Reply:
I like "revise"! But I'm more concerned about consistency than any particular choice of verb, so go with whatever you prefer.

Reply:
I went with "revise" in 966e1ad.